pyproject.toml

```diff
@@ -32,12 +32,37 @@ exclude = [
 ]
 
 [project.optional-dependencies]
-cudf = ["cudf>=23.08.00"]
+dev = [
+    "covdefaults>=2.3.0",
+    "duckdb>=1.1.1",
+    "hypothesis>=6.112.1",
+    "pandas>=2.0.3",
+    "polars>=1.8.1",
+    "pre-commit>=3.5.0",
+    "pyarrow>=17.0.0",
+    "pytest>=8.3.3",
+    "pytest-cov>=5.0.0",
+    "pytest-env>=1.1.5",
+    "pytest-randomly>=3.15.0",
+    "scikit-learn>=1.3.2",
+    "tqdm>=4.66.5",
+    "typing-extensions>=4.12.2",
+    "dask[dataframe]>=2023.5.0"
+]
+docs = [
+    "jinja2>=3.1.4",
+    "markdown-exec[ansi]>=1.9.3",
+    "mkdocs>=1.6.1",
+    "mkdocs-material>=9.5.36",
+    "mkdocstrings[python]>=0.26.1",
+    "polars>=1.0.0",
+]
+#cudf = ["cudf>=23.08.00"]
```

Comment:
Any reason for commenting this one out? 🤔

Reply:
Hi, sorry for the delayed response. If I recall correctly, @MarcoGorelli brought up this issue regarding installing cuDF with conda. The installation process is quite complex, and I'm not sure how to resolve it.
<img width="1013" alt="image" src="https://github.com/user-attachments/assets/d63bb215-225a-4251-b1bc-bacb79f1d202">